compiler/one-cmds/tests/prepare_test_materials.sh

```diff
@@ -68,7 +68,7 @@ if [[ ! -s "inception_v3_test_data.h5" ]]; then
 fi
 
 if [[ ! -d "test_saved_model" ]]; then
-    rm -rf test_saved_model.zip
+    rm -rf test_saved_model.zipo
```

Comment:
?

Reply:
Oh.. I am sorry that i made a mistake. 😅
Thank you a lot!